types/dygraphs/index.d.ts

```diff
@@ -900,57 +1106,68 @@ declare namespace dygraphs {
          */
         icon?: string;
 
-        /**    Width (in pixels) of the annotation flag or icon. */
+        /** Width (in pixels) of the annotation flag or icon. */
         width?: number;
+
         /** Height (in pixels) of the annotation flag or icon. */
         height?: number;
 
-        /**    CSS class to use for styling the annotation. */
+        /** CSS class to use for styling the annotation. */
         cssClass?: string;
 
-        /**    Height of the tick mark (in pixels) connecting the point to its flag or icon. */
+        /** Height of the tick mark (in pixels) connecting the point to its flag or icon. */
         tickHeight?: number;
 
-        /**    If true, attach annotations to the x-axis, rather than to actual points. */
-        attachAtBottom?: boolean;
+        /** Width of the tick mark connecting the point to its flag or icon. */
+        tickWidth?: number;
 
-        div?: HTMLDivElement;
```

Comment:
While you can't specify `div` when you create an annotation, annotations do have this property when they're passed to the various annotation callbacks. See https://dygraphs.com/tests/annotation.html for an example of it in use.

The distinction between input & output annotations is probably not worth the hassle to model, so I think it's reasonable to keep `div?: HTMLDivElement` here.

Reply:
Thanks. I will give it back.